docs/doc/30-reference/30-sql/40-show/show-tables.md

```diff
@@ -34,46 +34,61 @@ mysql> SHOW TABLES;
 
 Showing the tables with table name `"numbers_local"`:
 ```sql
-mysql> SHOW TABLES LIKE 'numbers_local';
-+---------------+
-| name          |
-+---------------+
-| numbers_local |
-+---------------+
+mysql> SHOW TABLES LIKE 'settings';
++------------+
+| table_name |
```

Comment:
Not `Tables_in_default` ?

Reply:
No, just `show full table` will modify the col name.